notion_client/helpers.py

```diff
@@ -24,3 +24,76 @@ def get_id(url: str) -> str:
         raise ValueError("The path in the URL seems to be incorrect.")
     raw_id = path[-32:]
     return str(UUID(raw_id))
+
+
+def iterate_paginated_API(
+    function: Callable[..., Any], **kwargs: Any
+) -> Generator[List[Any], None, None]:
+    """Return an iterator over the results of any paginated Notion API."""
+    next_cursor = None
+
+    while True:
+        response = function(kwargs, start_cursor=next_cursor)
```

Comment:
Is this not supposed to be `**kwargs`? Awaiting the merge of this PR, I copied the code but I had to change this to make it work.